privacy-toolset.php

```diff
@@ -9,20 +9,59 @@
  * @package automattic-privacy-toolset
  */
 
-// TODO setup plugin class
-// TODO standardize function names
-// TODO add noscript GTM code in body
-// TODO better version than current time
+namespace Automattic\PrivacyToolset;
 
-function create_privacy_toolset_block() {
-  register_block_type(__DIR__ . '/block/build');
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
 }
-add_action('init', 'create_privacy_toolset_block');
 
-function privacy_toolset_enqueue_styles() {
-  wp_enqueue_style(
-    'privacy-toolset',
-    __DIR__ . '/block/build/privacy-consent-banner.css?ver=' . time()
-  );
+class PrivacyToolset {
+
+  	/**
+     * Javascript built asset path.
+     *
+     * @var string
+     */
+    private $javascript_path = 'block/build/privacy-consent-banner.js';
+
+    /**
+     * CSS built asset path.
+     *
+     * @var string
+     */
+    private $css_path = 'block/build/privacy-consent-banner.css';
+
+    /**
+	   * Class constructor.
+	   */
+    public function __construct() {
+        add_action( 'init', [ $this, 'register_assets' ] );
+    }
```

Comment:
The purpose of a class constructor is to set up the object. It should not have unrelated code, such as integrations with other parts of WordPress. Code like this means that it can't be used in tests, for instance, or instantiated by other code, without having side effects.

Rename this method to something like `init()`, and then call that method after you've instantiated the class into an object,